src/tests/example.spec.js

```diff
@@ -28,3 +24,80 @@ test.describe('Saucedemo app basic tests', () => {
+        ) => {
+            const inventoryInitialOrder = await app.inventory.inventoryItemName.allTextContents();
+            await app.inventory.sortDropdown.selectOption({ value: option.sort });
+            const actualOrder = await app.inventory.inventoryItemName.allTextContents();
+            const expectedOrder = option.sortByName(inventoryInitialOrder);
+            expect(actualOrder).toEqual(expectedOrder);
+        });
+    };
+
+    const sortByPriceOptions = [
+        { sort: "lohi", sortByPrice: (inventoryInitialOrder) => inventoryInitialOrder.sort((a, b) => a - b) },
+        { sort: "hilo", sortByPrice: (inventoryInitialOrder) => inventoryInitialOrder.sort((a, b) => b - a) }
+    ];
+
+    for (const option of sortByPriceOptions) {
+        test(`Inventory sorting by price: ${option.sort}`, async (
+            /** @type {{ app: import('../pages/Application').Application }} */ { app },
+        ) => {
+            const inventoryInitialOrder = await app.inventory.inventoryPrice.allTextContents();
+            const inventoryInitialOrderToNumbers = inventoryInitialOrder.map((str) => parseFloat(str.replace('$', '')));
+            await app.inventory.sortDropdown.selectOption({ value: option.sort });
+            const actualOrder = (await app.inventory.inventoryPrice.allTextContents()).map((str) => parseFloat(str.replace('$', '')));
+            const expectedOrder = option.sortByPrice(inventoryInitialOrderToNumbers);
+            expect(actualOrder).toEqual(expectedOrder);
+        });
+    };
+      
+   
+    test.only('adding to cart several products', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {    
+        const inventoryList = await app.inventory.getInventoryItemsList();
+        const amount = inventoryList.length;
+
+        const randomInventories = await app.inventory.calculateRandomIndex(amount, 2);
+        const addedInventoriesData = await app.inventory.addRandomInventoriesToCart(randomInventories);
+        await app.baseSwagLab.openCartPage();
+        const inventoryAddedToCart = await app.shoppingCart.getCartItemsList();
+
+        for (const i in addedInventoriesData) {
+            expect(addedInventoriesData[i].name).toEqual(inventoryAddedToCart[i].name);
+            expect(addedInventoriesData[i].description).toEqual(inventoryAddedToCart[i].description);
+            expect(addedInventoriesData[i].price).toEqual(inventoryAddedToCart[i].price);
+        }                  
+    
+    }); 
+
+    // test('checkout flow', async (
+    //     /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    // ) => {    
+    //     const addedProducts = await app.inventory.addRandomProductsToCart(2);    
+    //     await app.baseSwagLab.openCartPage();
+    //     await app.shoppingCart.openCheckoutPage();
+    //     await app.customerInfo.fillUserData();      
+
+                       
+    
+    // });                    
```

Comment:
Should be uncommented or deleted

Reply:
@AlinaZhykharieva  this part currently in progress